wfc3tools/meta.yaml

```diff
@@ -26,7 +26,7 @@ requirements:
     - scipy
     - python x.x
 source:
-    git_tag: 1.3.1
+    git_tag: 1.3.3
```

Comment:
gah, the difference between relic and astropy still trips me, to be clean, would you rather me redo the git tag and notes or just update to the vtag